docs/METRICS.md

```diff
@@ -1,47 +1,22 @@
+
 ## Firefox Screenshots Metrics
+*Last Update: 2017-03-21*
 
-A summary of the metrics Firefox Screenshots will record, and what we're looking for in those metrics.
+This document is a summary of the metrics Firefox Screenshots will record, how we're recording them, and what we're looking for in those metrics.  There are two main areas we'll look at:
 
-### Opt-out
+0. Metrics to tell us if Firefox Screenshots is affecting Firefox.  For instance, do Firefox Screenshots users engage longer with Firefox?
```

Comment:
We've been talking about that.  It sounds like you don't need to leave the set of users with the feature disabled for a super long time to get a decent result (longer is better, of course, but hey...).  If our rollout lasts a couple weeks, that's already useful data.  We might be able to keep a small portion out longer, but that schedule isn't finalized.